reana_client/cli/workflow.py

```diff
@@ -874,8 +874,16 @@ def workflow_logs(
     help="REANA specification file describing the workflow to "
     "execute. [default=reana.yaml]",
 )
+@click.option(
+    "--environments",
+    is_flag=True,
+    default=False,
+    help="Check all runtime environments specified in REANA "
+    "specification file describing the workflow to "
+    "execute.",
```

Comment:
Perhaps we don't need the "describing the workflow to execute" part? It seems a bit redundant.

Reply:
We can add "[default=False]", as the `--file` option.